apps/web/docs/update-patch.md

```diff
@@ -0,0 +1,19 @@
+# Update yarn patches
+
+You can find all patches that are currently applied inside the `.yarn/patches` directory. The name of the package that the patch applies to can be found in the file name of the patch.
+
+Following are the steps to update a patch in case an update to the dependency needs to be done:
+
+1. Go to the `package.json` of where the dependency is located and change the version manually 
+   1. e.g. `"next": "patch:next@15.1.2#../../.yarn/patches/next-npm-15.1.2-24e7411703.patch"` -> `"next": "15.2.3"`
+2. Run `yarn install` to update the dependency
+3. Run `yarn patch <package-name>` e.g. `yarn patch next@npm:15.2.3`
```

Comment:
Maybe we should add this to post-install.

Reply:
When we still used an external package to create patches we had to do this as part of the post-install.